Documentation/policy/language.rst

```diff
@@ -308,9 +308,13 @@ init
     security identity has not been resolved yet. This is typically only
     observed in non-Kubernetes environments. See section
     :ref:`endpoint_lifecycle` for details.
+health
+    The health entity represents the health endpoints, used to check cluster
+    connectivity health. Each node managed by Cilium hosts a health endpoint.
+    See `cluster_connectivity_health` for details on health checks.
```

Comment:
I think one of the rebases or the merge commit introduced this (and maybe some of the other changes below) by mistake. This block of text in particular is already in master.

Reply:
Removed spurious client additions.